laravel-app/app/Services/ProfileService.php

```diff
@@ -7,6 +7,13 @@
 
 class ProfileService
 {
+    /**
+     * Update the profile.
+     *
+     * @param User $user
+     * @param array<string, mixed> $data
+     * @return void
+     */
     public function updateProfile(User $user, array $data)
```

Comment:
Since they are using Laravel 10. Laravel already removed doc block and used return type declaration.

Docs:
https://laravel.com/docs/10.x/releases#types
https://www.php.net/manual/en/language.types.declarations.php

Demo:
https://www.reddit.com/r/laravel/comments/11328e0/quick_demo_of_the_laravel_10x_shift/